Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -150,6 +150,19 @@
   categories:
   - UplinkAllies
 
+- type: listing
+  id: UplinkRifleAK220
+  name: uplink-Light-Rifle-AK220-name
+  description: uplink-Light-Rifle-AK220-desc
+  productEntity: WeaponLightRifleAK220
+  discountCategory: rareDiscounts
+  discountDownTo:
+    Telecrystal: 4
```

```suggestion
    Telecrystal: 3
```
Цена минимальной скидки рассчитывается делением на 5 и округлением в большую сторону